docs/current/data-model-howto.md

```diff
@@ -403,12 +404,13 @@ NOTE: Only IPMI is supported at this time.
 Provides information for a single deployment/data center/etc.
 
 ```
-|   name   |                        description                        | required |   schema  | default |
-|----------|-----------------------------------------------------------|----------|-----------|---------|
-| id       | A self-assigned identifier (should be unique)             | yes      | string    |         |
-| name     | A human-readable name for this group                      | no       | string    | id      |
-| agent    | Details for establishing a connection to the site's agent | yes      | env.agent |         |
-| metadata | unstructured, site-specific information                   | no       | object    |         |
+|   name   |                        description                        | required |      schema      | default |
+|----------|-----------------------------------------------------------|----------|------------------|---------|
+| id       | A self-assigned identifier (should be unique)             | yes      | string           |         |
+| name     | A human-readable name for this group                      | no       | string           | id      |
+| agent    | Details for establishing a connection to the site's agent | yes      | env.agent        |         |
+| subnets  | List of subnets for this cluster                          | yes      | env.subnet array |         |
```

Comment:
Maybe call it "Subnet" since this is just a single subnet. It should be subnetID?